README.md

```diff
@@ -51,6 +51,7 @@ Once the SEMOASA format gets some traction and achieves a level of stability, we
         * **\[extensionPropertyName\]** (patterned field, type: Extension Object or reference)- name of the specification extension property. must start with "x-" as per the OAS specification. We may want to support wildcards or template parameters in the name. May contain an Extension Object or a JSON Reference thereto, supporting the _Directory Catalog_ use case (one level deeper than namespace). 
             * **summary** - a plain-text summary of the purpose of the extension. 
             * **description** - markdown documentation describing the extension. 
+            * **deprecated** - a Boolean to indicate whether this extension is deprecated and should no longer be used.
```

Comment:
Good idea.

I would add to the description: "If the extension property is deprecated, the `summary` or `description` SHOULD specify the reason, and/or point to another document defining an extension property that replaces the this one."

Reply:
👍 lgtm.